app/scripts/views/settings/clients.js

```diff
@@ -151,7 +151,20 @@ define(function (require, exports, module) {
       return this._attachedClients.fetchClients({
         devices: true,
         oAuthApps: true
-      }, this.user);
+      }, this.user).then(() => {
+        // log the number of items
+        const numOfClients = this._attachedClients.length;
+
+        if (numOfClients === 0) {
+          this.logViewEvent('items.zero');
+        } else if (numOfClients === 1) {
+          this.logViewEvent('items.one');
+        } else if (numOfClients === 2) {
+          this.logViewEvent('items.two');
+        } else {
+          this.logViewEvent('items.many');
```

Comment:
I think this might be ok, if we noticed a large number of `items.many` events, we could revise.